### PR TITLE
 Vert.x demo

### PR DESCRIPTION
**Pull request by jpechane**

### CODE REVIEW

File: end-to-end-demo/debezium-vertx-demo/src/main/java/io/debezium/example/endtoend/vertx/Verticle.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    private static final String CONF_TOPIC = "demo.topic";
+    private static final String CONF_PORT = "demo.port";
+
+    private static final Logger LOG = LoggerFactory.getLogger(Verticle.class);
+
+    @Override
+    public void start() throws Exception {
+        final Map<SocketAddress, ServerWebSocket> sockets = new ConcurrentHashMap<>();
+
+        config().put(CONF_BOOTSTRAP_SERVERS, "localhost:9092");
```
Comment (gunnarmorling) [inline end-to-end-demo/debezium-vertx-demo/src/main/java/io/debezium/example/endtoend/vertx/Verticle.java]: Indeed localhost? Not kafka?

**Review by gunnarmorling — COMMENTED** [review]
Very nice, @jpechane. One comment inline. And can you add instructions to the README.md, too? Thanks!

File: end-to-end-demo/debezium-vertx-demo/src/main/java/io/debezium/example/endtoend/vertx/Verticle.java (hunk 1 above)
Comment (jpechane) [reply end-to-end-demo/debezium-vertx-demo/src/main/java/io/debezium/example/endtoend/vertx/Verticle.java]: Good catch! I am using an externa config property so I forgot to change the default value.

**Comment by jpechane** [comment]
@gunnarmorling Docs added

**Comment by gunnarmorling** [comment]
Merged; thanks, @jpechane!

**Pull request merged**